src/helix-ui/styles/modules/choice-tiles.less

```diff
@@ -7,8 +8,20 @@
 // Layout & Geometry
 /* ========== COMMON ========== */
 label.hxChoice > input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  border: 0;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
   position: absolute;
-  top: -200px;
+  width: 1px;
+
+  &:focus {
+    #Util.resetFocus();
```

Comment:
Remove the focus ring from the "invisible" input located behind the top left corner of the choice tile.